Base/res/html/misc/welcome.html

```diff
@@ -50,6 +50,8 @@ <h1>Welcome to the Serenity Browser!</h1>
     <p>Your user agent is: <b><span id="ua"></span></b></p>
     <p>This page loaded in <b><span id="loadtime"></span></b> ms</p>
     <nav class="page-list">
+        <ul>
+        </ul>
```

Comment:
What's this for?

Reply:
That's a leftover that does not render so I did not see it :( Sorry for that.